### ISSUE TITLE
PropertyPlot doesn't plot correctly

### ISSUE THREAD

**Issue by romarro** [CoolProp/CoolProp#1138]
Hi I am trying to use PropertyPlot from CoolProp 6.0.0 with WinPython 3.5.1.3 but it doesn't plot correctly, as you can see on the picture attached and I also get a lot of warnings:
```
CoolProp ver:6.0.0
C:\WinPython-64bit-3.5.1.3\python-3.5.1.amd64\lib\site-packages\CoolProp\Plots\Common.py:492: UserWarning: Please use "calc_sat_range" to calculate saturation and isoquality lines. Input ranges are discarded.
  UserWarning)
C:\WinPython-64bit-3.5.1.3\python-3.5.1.amd64\lib\site-packages\CoolProp\Plots\Common.py:520: UserWarning: An error occurred for inputs 8919480.287955, 2423.071691 with index (248,): unable to solve 1phase PY flash with Tmin=169.849, Tmax=682.5 due to error: HSU_P_flash_singlephase_Brent could not find a solution because Smolar [247.231 J/mol/K] is above the maximum value of 247.210601809 J/mol/K
  UserWarning)
C:\WinPython-64bit-3.5.1.3\python-3.5.1.amd64\lib\site-packages\CoolProp\Plots\Common.py:520: UserWarning: An error occurred for inputs 9133379.086438, 2423.071691 with index (249,): unable to solve 1phase PY flash with Tmin=169.849, Tmax=682.5 due to error: HSU_P_flash_singlephase_Brent could not find a solution because Smolar [247.231 J/mol/K] is above the maximum value of 246.954143497 J/mol/K
  UserWarning)
C:\WinPython-64bit-3.5.1.3\python-3.5.1.amd64\lib\site-packages\CoolProp\Plots\Plots.py:221: UserWarning: Detected an incomplete phase envelope, fixing it numerically.
  warnings.warn("Detected an incomplete phase envelope, fixing it numerically.")
```
The code used is this:
``` python
import CoolProp
from CoolProp.Plots import PropertyPlot

print('CoolProp ver:%s'%(CoolProp.__version__))
plot = PropertyPlot('HEOS::R134a', 'PH', unit_system='EUR', tp_limits='ACHP')
plot.calc_isolines(CoolProp.iQ, num=11)
plot.calc_isolines(CoolProp.iT, num=25)
plot.calc_isolines(CoolProp.iSmass, num=15)
plot.show()
```
and the resulted plot is:

![tst](https://cloud.githubusercontent.com/assets/7783693/16520510/16831374-3f99-11e6-843e-0b03f0e363e6.png)

**Comment by ibell** [comment]
@jowr?

**Comment by jowr** [comment]
Please update your code, with the latest version from the master branch I get this graphs:
![figure_1](https://cloud.githubusercontent.com/assets/769593/16713461/a2f90924-46a8-11e6-9ee2-31dca18b66db.png)

**Comment by JaimeDMN** [comment]
Please someone could tell me if this topic really is solved and if so how to solve it. I am currently with CoolProp ver: 6.0.0
```python
import CoolProp
from CoolProp.Plots import PropertyPlot

print('CoolProp ver:%s'%(CoolProp.__version__))
plot = PropertyPlot('HEOS::R134a', 'PH', unit_system='EUR', tp_limits='ACHP')
plot.calc_isolines(CoolProp.iQ, num=11)
plot.calc_isolines(CoolProp.iT, num=25)
plot.calc_isolines(CoolProp.iSmass, num=15)
plot.show()

```
```erro
c:\anaconda3\lib\site-packages\CoolProp\Plots\Common.py:492: UserWarning: Please use "calc_sat_range" to calculate saturation and isoquality lines. Input ranges are discarded.
  UserWarning)
c:\anaconda3\lib\site-packages\CoolProp\Plots\Common.py:520: UserWarning: An error occurred for inputs 8919480.287955, 2423.071691 with index (248,): unable to solve 1phase PY flash with Tmin=169.849, Tmax=682.5 due to error: HSU_P_flash_singlephase_Brent could not find a solution because Smolar [247.231 J/mol/K] is above the maximum value of 247.210601809 J/mol/K
  UserWarning)
c:\anaconda3\lib\site-packages\CoolProp\Plots\Common.py:520: UserWarning: An error occurred for inputs 9133379.086438, 2423.071691 with index (249,): unable to solve 1phase PY flash with Tmin=169.849, Tmax=682.5 due to error: HSU_P_flash_singlephase_Brent could not find a solution because Smolar [247.231 J/mol/K] is above the maximum value of 246.954143497 J/mol/K
  UserWarning)
c:\anaconda3\lib\site-packages\CoolProp\Plots\Plots.py:221: UserWarning: Detected an incomplete phase envelope, fixing it numerically.
  warnings.warn("Detected an incomplete phase envelope, fixing it numerically.")

```

**Comment by ibell** [comment]
@jowr any idea?

**Comment by jowr** [comment]
As I said, I fixed this some time ago, but after the release of 6.0.0. Your code will work with the next version of CoolProp. Until then, you can use the development version either by installing with pip (second option in http://coolprop.sourceforge.net/coolprop/wrappers/Python/index.html#automatic-installation) or you compile from source http://coolprop.sourceforge.net/coolprop/wrappers/Python/index.html#manual-installation 

In short: update CoolProp to the unreleased bleeding edge version and it will work.

**Comment by W2009** [comment]
Solution does not work for me. I have just installed latest revision CoolProp-6.1.1.dev0 (python: Python 3.6.3) and still plot is incorrect and the same error code.

Could you help with that issue?

**Comment by myjesus123** [comment]
recently, I have also  same figuration for coolprop graph. 
In my case, it was solved by re-install matplotlib and coolprop module.

**Comment by stathismast** [comment]
I have the same exact problem. I tried reinstalling matplotlib and coolprop, I tried using the nightly build but nothing worked. Anyone found a different solution?

**Comment by ibell** [comment]
Hi folks, the sad reality is that the plotting in CoolProp needs some
love.  I can't do it, but somebody needs to step in and fix the plotting
routines, which are good when they work, but currently broken.

**Comment by BlueEis** [comment]
3 years later... is there any solution for this problem?
I have it now.
CoolProp 6.4.1
Python 3.8
Matplotlib 3.5.1
Numpy 1.22.2

**Comment by aurelien14** [comment]
You must install scipy because there are interpolations to build curves
